.azure-pipelines/ultimate-pipeline.yml

```diff
@@ -2700,7 +2700,8 @@ stages:
         ./run-appsec.sh "linux"
       displayName: Crank
       env:
-        DD_SERVICE: dd-trace-dotnet-appsec
+        DD_SERVICE: dd-trace-dotnet
```

Comment:
I assume the previous name wasn't used anywhere, could be worth a check with @robertpi

Reply:
Oops, just merged... Heads up @robertpi! 🙈